.github/buildomat/jobs/build-and-test.sh

```diff
@@ -14,7 +14,7 @@ cargo --version
 rustc --version
 
 banner build
-ptime -m cargo build --locked --all-targets --verbose
+ptime -m cargo build --all-features --locked --all-targets --verbose
```

Comment:
So this looks like it'll only test on illumos with the `usdt-probes` feature, whereas the GitHub workflow below tests both with and without that feature. Is that intentional? Did you want another run of `cargo build` / `cargo test` but without `--all-features`?

Reply:
Yeah, that's accurate.  It's not a strong feeling.  It just felt like I was already adding a lot more tests and I wasn't sure how necessary it was, given that we have the non-feature tests on other systems.